testsuite/module-publish/src/packages/simple/sources/Simple.move

```diff
@@ -57,12 +58,55 @@ module 0xABCD::simple {
     // Test simple CPU usage. Loop as defined by the input `count`.
     // Not a true test of CPU usage given the number of instructions
     // used, but a simple reference to computation with no data access.
-    public entry fun loopy(_s: &signer, count: u64) {
+    public entry fun loop_nop(_s: &signer, count: u64) {
         while (count > 0) {
             count = count - 1;
         }
     }
 
+    public entry fun loop_arithmetic(_s: &signer, count: u64) {
+        let a;
+        let b = 0;
+        let c;
+        let d = 0;
+        while (count > 0) {
+            count = count - 1;
+
+            a = b + 1;
```

Comment:
I was wondering if we should have another one to test * and /.

Reply:
we have a lot of different execution-related constants, proper gas calibrator is meant to tackle those